proto3-suite.cabal

```diff
@@ -67,7 +67,7 @@ library
 
   if flag(large-records)
     -- large-records support uses newer Dhall APIs. So we need at least 1.34.
-    build-depends:     dhall >=1.34 && < 1.39,
+    build-depends:     dhall >=1.34 && < 1.41.2,
```

Comment:
swagger2 >= 2.8 requires aeson 2, this conflicted with dhall <1.39 which requires pre-2 aeson. Bumped the upper bound to allow the solver to grab aeson 2.